app/src/androidTest/java/com/github/h3lp3rs/h3lp/H3lpAppTest.kt

```diff
@@ -116,6 +116,7 @@ open class H3lpAppTest {
 
         const val SWISS_LAT = 46.514
         const val SWISS_LONG = 6.604
+        const val MAX_RESPONSE_DISTANCE = 5000.0
```

Comment:
That would be  nice that the user can choose this parameters in settings.

This could be part of a new user story,

Reply:
Yes, thought of the same thing. Could be set in the preferences.